pkg/verifier/notation/truststore.go

```diff
@@ -98,11 +100,18 @@ func (s *trustStore) getCertificatesInternal(ctx context.Context, storeType trus
 			return certs, fmt.Errorf("no certificates fetched in namedStore: %+v", namedStore)
 		}
 	} else {
+		crlFetcher, err := s.crlHandler.NewFetcher()
```

Comment:
I'm thiniking if it's necessary to fetch the crl during getCertificate. Notation verify API will fetch the CRL during verification, do we need to do another Fetch?

Reply:
This is a `Pre-loading` stage, as we have the verifier config, including certs loaded from path, but not start verifying in non-cli scenarios.

Reply:
will `GetCertificates` will be invoked during pre-loading stage?

Reply:
mark resolved as discussed offline